package.json

```diff
@@ -44,8 +45,11 @@
     "@types/web3": "^1.0.19",
     "@typescript-eslint/eslint-plugin": "^1.13.0",
     "@typescript-eslint/parser": "^1.13.0",
+    "babel-jest": "^24.9.0",
     "babel-loader": "^8.0.6",
     "babel-plugin-styled-components": "^1.10.6",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
```

Comment:
Where did you use enzyme?

Reply:
Nowhere yet. It's supposed to be used to test components. In case we don't, this might be unnecessary. Would you like me to remove that?

Reply:
I think it's better to include it in a separate PR, just with a basic render of a component

Reply:
Removed in latest commit.